e2e/framework/manager/rook_test_infra.go

```diff
@@ -196,8 +198,24 @@ func (r *rookTestInfraManager) copyImageToNode(containerId string, imageName str
 	return nil
 }
 
+func (r *rookTestInfraManager) configureSetupStatus(status bool) {
```

Comment:
if you're just going to set a flag, i think the calling code would be more readable if you didn't have these one-line helper methods `configureSetupStatus` and `confugreRookInstalledStatus`. Just assign `r.isSetup = true` directly.